pkg/assets/job.yaml.tpl

```diff
@@ -260,11 +260,11 @@ spec:
         {{- end }}
         resources:
           limits:
-            cpu: 1
-            memory: 1Gi
+            cpu: {{ .DefaultExecutorCPULimit }}  
```

Comment:
Perhaps leave off the CPU default limit (make it conditional, so if set to a custom value only then will it be specified in the template).

Reply:
Do we prefer https://github.com/appvia/terranetes-controller/pull/1501/commits/fec8d22d5b8a83bb429d15288b0851a054f736e3 .. and place the decision solely on the administrator? cc @KashifSaadat

Reply:
sorry rebased - https://github.com/appvia/terranetes-controller/commit/9ddbaafa6e6ce1e1f7b07a096f560a3fafcbe463